docker/Dockerfile

```
@@ -0,0 +1,10 @@
+FROM nodered/node-red:latest
+
+USER node-red
+
+WORKDIR /usr/src/node-red
+
+COPY --chown=node-red:node-red package.json ./node-red-contrib-edgex-connector/
+COPY --chown=node-red:node-red Edgex_Node ./node-red-contrib-edgex-connector/Edgex_Node
+
+RUN npm install --unsafe-perm --no-update-notifier --no-fund ./node-red-contrib-edgex-connector
```

`--unsafe-perm` should not be necessary here - you're already a non-root user.

```suggestion
RUN npm install --no-update-notifier --no-fund ./node-red-contrib-edgex-connector
```